cli/kata-check_test.go

```diff
@@ -121,10 +122,17 @@ func makeCPUInfoFile(path, vendorID, flags string) error {
 	if err != nil {
 		return err
 	}
-
-	args := map[string]string{
-		"Flags":    flags,
-		"VendorID": vendorID,
+	var args map[string]string
+	if goruntime.GOARCH == "s390x" {
```

Comment:
I'd rather we don't add architecture-specific checks like this and instead do one of:

- Make `makeCPUInfoFile()` architecture-specific.
  - Rename this function `genericMakeCPUInfoFile()`.
  - Add a `makeCPUInfoFile()` to each `kata-check_${GOARCH}_test.go` file.
  - Make the `makeCPUInfoFile()` functions call `genericMakeCPUInfoFile()`, but for 390 have a custom implementation.
- Define the `args` map as in each `kata-check_${GOARCH}_test.go` file. Again, we could make each architecture use a generic map or they can provide their own custom implementation.